src/locale/locales/de/messages.po

```diff
@@ -1350,35 +1292,31 @@ msgstr "Gib die E-Mail ein, die du zur Erstellung deines Kontos verwendet hast.
 #: src/view/com/auth/create/Step1.tsx:251
 #: src/view/com/modals/BirthDateSettings.tsx:74
 msgid "Enter your birth date"
-msgstr ""
+msgstr "Gib deine Geburtsdatum ein"
```

Comment:
```suggestion
msgstr "Gib dein Geburtsdatum ein"
```

btw funny how deepl is switching beween *du (dein)* and *sie*

Reply:
I think this might have been a tweak from me actually, because I noticed that other similar phrases had been translated previously as "Gib dein/e"